Microsoft.WindowsAzure.Storage/includes/was/file.h

```diff
@@ -5079,7 +5080,9 @@ namespace azure { namespace storage {
             {
                 throw std::runtime_error("Cannot access a cloud file directory as cloud file");
             }
-            return cloud_file(m_name, m_directory);
+            cloud_file result = cloud_file(m_name, m_directory);
+            result.properties()->m_length = std::static_cast<size_t> m_length;
```

Comment:
Why are we static-casting the value to `size_t` here?

And also according to [here](https://en.cppreference.com/w/cpp/language/static_cast), `static_cast` does not come with `std::` and there are parentheses.